### DISCUSSION TITLE
Tag Drag and drop / Move [Help Wanted]

### DISCUSSION THREAD

**Discussion by OA971** [standardnotes/forum#2932] [General] [answered]
I moved a main Tag by mistake to another tag and it became its sub.
How can I  make it a main tag again?

**Comment by effieeee** [comment, marked as answer]
Hi @OA971 , you will need to drag it down to the bottom of your tag list, where there will be an available space to remove the tag from its current folder hierarchy

![image](https://user-images.githubusercontent.com/54989748/208698622-2b0b247c-6500-44f5-a17f-09e9560c09c0.png)

Of course, this is not ideal for users with very long lists of tags, so we'll keep it in mind to have an easier way to do this 🙏
